suite/tests/CMakeLists.txt

```diff
@@ -2537,7 +2537,10 @@ endif ()
   endif ()
 
   if (X86) # Tests x86 reachability.
-    tobuild_ci(client.reachability client-interface/reachability.c "" "" "")
+    tobuild_ci(client.reachability client-interface/reachability.c ""
+      # The vm_base options were added in order to expose bugs that fail to xl8 reachable
+      # addresses to the executable view within the scheme of satisfy_w_xor_x.
+      "-no_vm_base_near_app -vm_base 0x100000000" "")
```

Comment:
Value too large for 32-bit.

Reply:
I took the value from existing tests (that are enabled in 32-bit). Changed it for this one to 0x10000000. Done.

Reply:
Is 0x10000000 going to work?  It seems too close to a non-PIE app.  I thought the original addr was good for 64-bit.  Maybe just don't pass any options for 32-bit.

Reply:
It worked meaning it passed testing. Ok, changing it back, this time splitting X64 and X86.